site/js/index.js

```diff
@@ -17,20 +18,33 @@ const main = async function() {
     modals:   new ModalsActor,   // Creates and handles modal dialogs.
   }
 
-  // Establish a WebSocket connection. It's *almost* an
-  // actor but not enough to live in the `actors` object.
-  const socket = io()
-
-  // Actors get references to other actors, plus a
-  // reference to the WebSocket connection.
+  // Actors get references to other actors.
   for (const [ name, actor ] of Object.entries(actors)) {
     actor.name = name
     actor.actors = actors
-    actor.socket = socket
 
-    actor.init() // Actors should subscribe to events here.
+    actor.init() // Actors should subscribe to events from eachother here.
```

Comment:
Typo? :package:

Reply:
Typo?

Reply:
"Eachother" should be two words - "each other".

Reply:
o

Reply:
@towerofnix can you fix this 📦